internals/export/elasticsearch.go

```diff
@@ -116,9 +116,13 @@ func (export StreamedExport) ProcessStreamedExport(ctx context.Context, params E
 	}
 
 	// handle pit creation
-	pit, err := cli.OpenPointInTime(params.Indices).
-		IgnoreUnavailable(params.IgnoreUnavailable).
-		KeepAlive("5m").Do(context.Background())
+	pitRequest := cli.OpenPointInTime(params.Indices).KeepAlive("5m")
+	if params.IgnoreUnavailable {
+		pitRequest = pitRequest.IgnoreUnavailable(true)
+	}
```

Comment:
![image](https://github.com/myrteametrics/myrtea-engine-api/assets/8338662/6b37d475-ad17-4747-9546-b5f625a8cb30)
default is false so you can stay on the old version